doc/orchestrator.rst

```diff
@@ -1,207 +1,695 @@
+`db-identifier` and ``ConfigOptions`` object specified at ``Orchestrator`` initialization time.
+
+-------
+Example
+-------
+In the following example, we demonstrate deploying a standalone orchestrator on an HPC System.
+Once the standalone orchestrator is launched from the driver script, we walk through
+connecting a SmartRedis client to the orchestrator from within the SmartSim model
+script to transmit data then poll for the existence of the data.
 
+The example is comprised of two script files:
 
-Cluster Orchestrator
-====================
+- :ref:`Application Script<standalone_orch_app_script>`
+   The application script is a Python file that contains instructions to create a SmartRedis
+   client connection to the standalone orchestrator launched in the driver script.
+   To demonstrate the ability of workflow components to access data from
+   other entities, we then retrieve the tensors set by the driver script using a SmartRedis client in
+   the application script. We then instruct the client to send and retrieve data from within the application script.
+- :ref:`Experiment Driver Script<standalone_orch_driver_script>`
+   The experiment driver script is responsible for launching and managing SmartSim entities. Within this script,
+   we use the Experiment API to create and launch a standalone orchestrator. To demonstrate the capability of
+   SmartSim model applications to access orchestrator data sent from other sources, we employ the SmartRedis ``Client`` in
+   the driver script to store a tensor in the orchestrator, which is later retrieved by the SmartSim model.
+   To employ the application script, we initialize a ``Model`` object with the application script as an executable argument,
+   launch the ``Orchestrator``, and then launch the ``Model``.
+
+   To further demonstrate the ability of workflow components to access data from
+   other entities, we then retrieve the tensors stored by the ``Model`` using a SmartRedis client in
+   the driver script. Lastly, we tear down the ``Orchestrator``.
+
+.. _standalone_orch_app_script:
+Application Script
+==================
+To begin writing the application script, import the necessary SmartRedis packages:
 
-The ``Orchestrator`` supports single node and distributed memory settings. This means
-that a single compute host can be used for the database or multiple by specifying
-``db_nodes`` to be greater than 1.
+.. code-block:: python
 
-.. |cluster-orc| image:: images/clustered-orc-diagram.png
-  :width: 700
-  :alt: Alternative text
+  from smartredis import Client, log_data
+  from smartredis import *
+  import numpy as np
 
-|cluster-orc|
+Client Initialization
+---------------------
+To establish a connection with the orchestrator, we need to initialize a new SmartRedis client.
+Since the ``Orchestrator`` we launch in the driver script is sharded, we specify the
+constructor argument `cluster` as `True`.
 
+.. note::
```

Comment:
Maybe put all of the notes after the code block.  The code block sort of gets lost in the rendered version